starbucks_dw/models/reporting/report_channel_effectiveness.sql

```diff
@@ -0,0 +1,42 @@
+{{
+  config(
+    materialized = 'table',
+    )
+}}
+
+WITH prep_offer_response AS (
+
+SELECT
+offer_channel,
+offer_type,
+{{ dbt_utils.pivot(
+      'transaction_status',
+      dbt_utils.get_column_values(ref('fct_offer_transactions'), 'transaction_status')
+) }}
+FROM {{ ref('fct_offer_transactions') }} fct
+WHERE offer_type!='informational'
+GROUP BY offer_channel, offer_type
+),
+all_aggs AS (
+
+SELECT
+offer_channel,
+'all' as offer_type,
+{{ dbt_utils.pivot(
+      'transaction_status',
+      dbt_utils.get_column_values(ref('fct_offer_transactions'), 'transaction_status')
```

Comment:
The same thing here:

```suggestion
      dbt_utils.get_column_values(table=ref('fct_offer_transactions'), column='transaction_status')
```